pkgs/development/tools/pre-commit/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pythonPackages }:
+with pythonPackages; buildPythonApplication rec {
+  pname = "pre_commit";
+  version = "1.10.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kn8h9k9ca330m5n7r4cvxp679y3sc95m1x23a3qhzgam09n7jwr";
+  };
+
+  propagatedBuildInputs = [
+    aspy-yaml
+    cached-property
+    cfgv
+    identify
+    nodeenv
+    six
+    toml
+    virtualenv
+  ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests? Please add a comment.

Reply:
The tests fail on python3 due to a windll dependency. I will add a comment explaining, I should have done that right away :sweat_smile: